src/lib/index.tsx

```diff
@@ -25,6 +25,11 @@ const containerBaseStyles: React.CSSProperties = {
   position: "relative",
 };
 
+const light: React.CSSProperties = {
+  backgroundColor: "white",
+  color: "black",
+}
+
 const tooltipBaseStyles: React.CSSProperties = {
```

Comment:
Make this a function that accepts theme instead of creating another variable.

Reply:
You mean a function that takes theme as input and returns the properties of the theme.
in this case, if theme is light function should return {backgrondColor: "white", color: "black"}

Reply:
Yes check #12, also create a type for theme similar to #12

Reply:
Done! :)